playground/play002.ts

```
@@ -0,0 +1,66 @@
+  JSONParseError
+> = Effect.try({
+  try: () =>
+    JSON.parse(
+      `[{"title": "iphone", "price": 100}, {"title": "samsung", "price": "ciao"}]`
+    ),
+  catch: (unknownError) =>
+    new JSONParseError(`JSON parsing failed: ${unknownError}}`),
+});
+
+const mul = (p: number): Effect.Effect<number, number> => {
+  if (!isNaN(p)) return Effect.succeed(p * 2);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)